src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/AddAStandard/ViewTrainingLocationsController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Authorization.Mvc.Attributes;
+using SFA.DAS.Roatp.CourseManagement.Web.Infrastructure;
+using SFA.DAS.Roatp.CourseManagement.Web.Infrastructure.Authorization;
+using SFA.DAS.Roatp.CourseManagement.Web.Models.AddAStandard;
+using SFA.DAS.Roatp.CourseManagement.Web.Models.ProviderCourseLocations;
+using SFA.DAS.Roatp.CourseManagement.Web.Services;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.Controllers.AddAStandard
+{
+    [DasAuthorize(new[] { "ProviderFeature.CourseManagement" }, Policy = nameof(PolicyNames.HasProviderAccount))]
+
+    public class ViewTrainingLocationsController : AddAStandardControllerBase
+    {
+        public const string ViewPath = "~/Views/AddAStandard/ViewTrainingLocations.cshtml";
```

Comment:
View name should be StandardTrainingLocations.cshtml

Reply:
done